docs/docs/features/endpoints.md

```diff
@@ -12,7 +12,8 @@ OAuth2 Proxy responds directly to the following endpoints. All other endpoints w
 - /oauth2/start - a URL that will redirect to start the OAuth cycle
 - /oauth2/callback - the URL used at the end of the OAuth cycle. The oauth app will be configured with this as the callback url.
 - /oauth2/userinfo - the URL is used to return user's email from the session in JSON format.
-- /oauth2/auth - only returns a 202 Accepted response or a 401 Unauthorized response; for use with the [Nginx `auth_request` directive](../configuration/overview.md#configuring-for-use-with-the-nginx-auth_request-directive)
+- /oauth2/auth - only returns a 202 Accepted response or a 401 Unauthorized response; supports [Nginx `auth_request` directive](../configuration/overview.md#configuring-for-use-with-the-nginx-auth_request-directive) and [Traefik (v2) `ForwardAuth` middleware](../configuration/overview.md#configuring-for-use-with-the-traefik-v2-forwardauth-middleware)
```

Comment:
So the use cases for `/oauth2/auth` 202/401 (and 403 too now I think?) have grown well beyond these two use cases. (Kubenetes ingress annotations, nginx, traefik, direct auth checks from applications, etc).

I think we should simplify this endpoint documentation and not call out specific use cases anymore (just mention the generic utility of subrequest auth checks).

Reply:
Have removed the Nginx and Traefik specific statements and reworded. Please do suggest if you would like me to rephrase